pkg/aws/client.go

```diff
@@ -484,6 +509,19 @@ func (c *awsClient) CheckAdminUserNotExisting(userName string) (err error) {
 	return nil
 }
 
+func (c *awsClient) CheckAdminUserExists(userName string) (err error) {
+	userList, err := c.iamClient.ListUsers(&iam.ListUsersInput{})
+	if err != nil {
+		return err
+	}
+	for _, user := range userList.Users {
+		if *user.UserName == userName {
+			return nil
```

Comment:
I think this should `return true, nil`

Reply:
Update to getUser and we get 'NoSuchEntity' exception. So used the error to identify whether it exists or not instead of extra bool variable.